package.json

```diff
@@ -11,7 +11,7 @@
   },
   "devDependencies": {
     "@commitlint/cli": "^6.1.2",
-    "@indec/react-native-commons": "^0.1.6",
+    "@indec/react-native-commons": "indec-it/react-native-commons",
```

Comment:
we should use a published version

Reply:
indec-it/react-native-common is ready to publish a version